src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1325,4 +1336,9 @@ private function isValidHtml5Heading(string $heading): bool
     {
         return 1 === preg_match('/^\x{FEFF}?\s*(<!--[^>]*?-->\s*)*$/u', $heading);
     }
+
+    private function clearCachedNamespaces(): void
```

Comment:
this method is used once and contains a single line of code. It content could be moved to `clean` method